monai/networks/nets/torchvision_fc.py

```diff
@@ -0,0 +1,43 @@
+import torch
+
+from monai.utils import optional_import
+
+models, _ = optional_import("torchvision.models")
+
+
+class TorchVisionFullyConvModel(torch.nn.Module):
+    """
+    Customize TorchVision models to replace fully connected layer by convolutional layer.
+
+    Args:
+        model_name: name of any torchvision with adaptive avg pooling and fully connected layer at the end.
+            Default to "resnet18".
```

Comment:
what are the supported options? perhaps provide a few examples other than `resnet18`